### PR TITLE
Add input_switch()

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

**Review by gadenbuie — COMMENTED** [review]
I left a few small comments and suggestions below.

I wonder if we should include arguments for `disabled` and `reverse` (to disable the input and to reverse the order of the `label` and `input`). I certainly see a lot of utility for `disabled`, but I think we'd also want to allow `update_switch()` to disable the input, which would require us to write some swtich-specific JS in bslib or to update the checkbox input binding in shiny. (I lean toward a bslib-specific approach.)

I'd be happy to pick up this PR and add both arguments.

**Comment by gadenbuie** [comment]
> I wonder if we should include arguments for `disabled` and `reverse` (to disable the input and to reverse the order of the `label` and `input`). I certainly see a lot of utility for `disabled`, but I think we'd also want to allow `update_switch()` to disable the input, which would require us to write some switch-specific JS in bslib or to update the checkbox input binding in shiny. (I lean toward a bslib-specific approach.)

@cpsievert and I discussed in person: this PR will focus on feature parity with Shiny for Python and we'll tackle the above ideas in a future PR

**Comment by gadenbuie** [comment]
@cpsievert I applied my suggestions and added a small example app. We could suggest [a similar app for Python](https://shinylive.io/py/editor/#code=NobwRAdghgtgpmAXGKAHVA6VBPMAaMAYwHsIAXOcpMAMwCdiYACAZwAsBLCbJjmVYnTJMAgujxMAkhFQBXMiwkB5eXIUSAynBYsOpCXUoATOHQmyOAHQjW0qAPoWmAXiYWsUAOZx7NADYWRgAU1kxhbhwYbABMIWAA0nDYAEbEUHRGTFpkZFyeLJZgAJR4oeHuXGr2LADuHGSEbHFQ8sT2hGj1UH4cAF5QuaSFEoUirQC0AMKdZN19A3o2+GEAKnSycCVlYRUy8tV1DU2FLWRtJHSGhINLI2BjZ1OCVzfDq+ubpRDhERiV+7V6o04o04IQANbVVBwPw9CCeN6FSZsMHgrLQ2F5N5rDZbb7lSL-MgHIHHMAsGDpYlyCDXWQLIbLQoaSlCJgABVktLI9NexS+P3cxFU+woAA9iQA3UzJBYwOKoQySjhwGqFPFFaxaiAmGisUzSuhBImIKR7dRMYVkNSmlTW+SKfU6RamrTO0hFRDbJgAAStam9PsMOtMGHFZG9uqYirgytVQU93p+hh5dG+ICTP3CJ1a7Rmc36fNNRIwp3O+Z6hcWCYFWbrObO7WeYKLvHNpdzFxe1bxdfrRBREKhMLhCKQbbUGFBQ5YGNHNcz-YpVPsNLpDKWxfby6Eq6565uCcXTAAvtrbOgXKJ0EE7I4OBIWAbTJqIGATwBdIA).

File: R/input-switch.R [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#' Create an on-off style switch control for specifying logical values.
+#'
+#' @examplesIf interactive()
+#' library(shiny)
+#' library(bslib)
+#'
+#' ui <- page_fixed(
+#'   title = "Keyboard Settings",
+#'   h2("Keyboard Settings"),
+#'   input_switch("auto_capitalization", "Auto-Capitalization",  TRUE),
```
Comment (cpsievert) [inline R/input-switch.R:12]:
```suggestion
#'   input_switch("auto_capitalization", "Auto-Capitalization", TRUE),
```

File: R/input-switch.R [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+#'   input_switch("auto_correction", "Auto-Correction",  TRUE),
```
Comment (cpsievert) [inline R/input-switch.R:13]:
```suggestion
#'   input_switch("auto_correction", "Auto-Correction", TRUE),
```

**Comment by cpsievert** [comment]
Here's [a slightly revised py-shiny version](https://shinylive.io/py/editor/#code=NobwRAdghgtgpmAXGKAHVA6VBPMAaMAYwHsIAXOcpMAMwCdiYACAZwAsBLCbJjmVYnTJMAgujxMAkhFQBXMiwkB5eXIUSAynBYsOpCXUoATOHQmyOAHQjW0qAPoWmAXiYWsUAOZx7NADYWRgAU1kxhbhwYbABMIWAA0nDYAEbEUHRGTFpkZFyeLJZgAJR4oeHuXGr2LADuHGSEbHFQ8sT2hGj1UH4cAF5QuaSFEoUirQC0AMKdZN19A3o2+GEAKnSycCVlYRUy8tV1DU2FLWRtJHSGhINLI2BjZ1OCVzfDq+ubpRDhERiV+7V6o04o04IQANbVVBwPw9CCeN6FSZsMHgrLQ2F5N5rDZbb7lSL-MgHIHHMAsGDpYlyCDXWQLIbLQoaSlCJgABVktLI9NexS+P3cxFU+woAA9iQA3UzJBYwOKoQySjhwGqFPFFaxaiAmGisUzSuhBImIKR7dRMYVkNSmlTW+SKfU6RamrTO0hFRDbJgAAStam9PsMOtMGHFZG9uqYirgytVQU93p+hh5dG+NEKmbAtla7Rmc36N1NICJGFO53zPULiwTAF8c2d2s8wUWmCXzWXcxcXjWivXaSiIVCYXDPMXS6ChywMaO69YKVT7DS6QyIOOOwuhEuuSubnP8T8wlnMzZT3YXKJ0EE7I4OBIWAbTJqIGBawBdIA)

**Comment by cpsievert** [comment]
Let's add a NEWS item then merge away!

**Pull request merged**